README.md

```diff
@@ -111,127 +78,95 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [coming soon](https://budget-ap.herokuapp.com/)
```

Comment:
- [ ] It will be great to ensure that the live version works. The app is not yet deployed.  Try and follow this [article](https://devcenter.heroku.com/articles/getting-started-with-rails7) on deploying the app to Heroku. If you are having problems with Heroku, try deploying it to render.com, use this [article](https://render.com/docs/deploy-rails)

Reply:
can you help me with the deploy?

Reply:
like a tutorial